docs/changelog.md

```diff
@@ -15,6 +15,10 @@ Way of initialization project from dictionary. Now it's possible as follows: Pro
 
 - Feature of initializing `peppy.Project` from `pandas.DataFrame`
 
+## [0.34.0] -- 2022-08-17
```

Comment:
Shouldn't be `0.35.0`? `0.34.0` is the current one

Reply:
I think, this mistake is after merging branches, I'll fix it